src/login/logind-session.h

```diff
@@ -204,8 +204,8 @@ SessionType session_type_from_string(const char *s) _pure_;
 const char* session_class_to_string(SessionClass t) _const_;
 SessionClass session_class_from_string(const char *s) _pure_;
 
-const char *kill_who_to_string(KillWho k) _const_;
-KillWho kill_who_from_string(const char *s) _pure_;
+const char *kill_whom_to_string(KillWhom k) _const_;
```

Comment:
```suggestion
const char* kill_whom_to_string(KillWhom k) _const_;
```

while at it?

Reply:
I updated with such a change. It touched a few more places, so please review again.